### DISCUSSION TITLE
✅ Import/Export list of SSH hosts

### DISCUSSION THREAD

**Discussion by Thunderhawk2001** [isontheline/pro.webssh.net#553] [Q&A] [answered]
Hi. 

Is there a way to import my current list of hosts on my pc? e.g. import from putty *.reg file or just from a text file?
An export function would also be great.

**Comment by isontheline** [comment, marked as answer]
Hi @Thunderhawk2001 👋

Yet there is no way to import / export from files (except [database backup](https://webssh.net/documentation/help/howtos/mashREPL/database-backup/)).
An issue is already opened here : https://github.com/isontheline/pro.webssh.net/issues/252

I need to find time to implement import / export, several users are waiting for it.

**Reply by Thunderhawk2001** [reply]
Ohh.. sorry. I didn't serach in the issues for this :(

But thank you for your fast reply.

**Reply by isontheline** [reply]
You're welcome @Thunderhawk2001 

I'm increasing the priority of the issue #252
